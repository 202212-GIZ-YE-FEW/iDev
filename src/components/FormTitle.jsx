import { useTranslation } from "next-i18next";

export default function FormTitle({ title }) {
    const { t } = useTranslation("common");
    return (
        <>
            <span className=' uppercase font-normal  md:text-xl lg:text-3xl  leading-75 text-center tracking-tighter'>
                {t(title)}
            </span>
        </>
    );
}
