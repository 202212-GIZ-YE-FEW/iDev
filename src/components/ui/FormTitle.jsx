import React from "react";

export default function FormTitle({ title }) {
    return (
        <>
            <span className='font-poppins  uppercase font-normal font-400 text-5xl leading-75 text-center tracking-tighter'>
                {title}
            </span>
        </>
    );
}
