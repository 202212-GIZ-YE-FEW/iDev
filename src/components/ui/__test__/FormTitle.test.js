import renderer from "react-test-renderer";

import FormTitle from "../FormTitle";

it("renders correctly", () => {
    const tree = renderer.create(<FormTitle />).toJSON();
    expect(tree).toMatchSnapshot();
});
