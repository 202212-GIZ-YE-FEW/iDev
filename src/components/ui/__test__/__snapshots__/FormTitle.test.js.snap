// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="font-poppins  uppercase font-normal font-400 text-5xl leading-75 text-center tracking-tighter"
/>
`;
